# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/rox-filer/install-location.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./install.sh.orig	2004-09-07 05:44:33.771458392 +0200
+++ ./install.sh	2004-09-07 05:47:11.450487536 +0200
@@ -94,10 +94,10 @@
 	     MANDIR=
 	   fi
 	   ;;
-	3) APPDIR=/usr/apps
-	   BINDIR=/usr/bin
+	3) BINDIR=/usr/bin
 	   SHAREDIR=/usr/share
-	   CHOICESDIR=${SHAREDIR}/Choices
+	   CHOICESDIR=${SHAREDIR}/ROX-Filer/Choices
+	   APPDIR=${SHAREDIR}/ROX-Filer/apps
 	   MANDIR=${SHAREDIR}/man
 	   ;;
 	4) echo "Where should the ROX-Filer application go?"
--- ./install.sh.orig	2003-04-29 12:56:22.000000000 +0200
+++ ./install.sh	2004-09-07 06:15:35.003508112 +0200
@@ -194,9 +194,9 @@
 endir "$MIMEDIR"
 endir "$MIMEDIR/packages"
 cp rox.xml "$MIMEINFO" || die "Failed to create $MIMEINFO"
-update-mime-database "$MIMEDIR" || die "update-mime-database failed
-Make sure you have installed the shared MIME database from:
-http://www.freedesktop.org/standards/shared-mime-info.html"
+# update-mime-database "$MIMEDIR" || die "update-mime-database failed
+# Make sure you have installed the shared MIME database from:
+# http://www.freedesktop.org/standards/shared-mime-info.html"
 
 echo "Installing application..."
 endir "$APPDIR"
